dali/operators/image/crop/bbox_crop.cc

```diff
@@ -27,7 +27,19 @@ labels match the boxes that remain, after being discarded with respect to the mi
 Be advised, when `allow_no_crop` is `false` and `thresholds` does not contain `0` it is good to increase `num_attempts` as otherwise
 it may loop for a very long time.)code")
     .NumInput(2)
+    .InputDoc(0, "BBoxes", "TensorList of floats",
+              "Bounding boxes to encode in `[l,t,r,b]` or `[x,y,w,h]` format")
+    .InputDoc(1, "Labels", "TensorList of int",
+              "Labels corresponding to bounding boxes")
     .NumOutput(4)
+    .OutputDoc(0, "Begin",
+               "TensorList of {batch, 2} floats", "Starting coordinate of crop ``(x, y)``")
```

Comment:
about to change as we'll support 3d as well (same below)

Reply:
Took changes from your PR.